core-services/release-controller/_releases/release-ocp-4.7-ci.json

```diff
@@ -67,6 +72,11 @@
       "interval":"6h",
       "upgrade":true,
       "prowJob":{"name":"release-openshift-origin-installer-e2e-gcp-upgrade-4.7"}
+    },
+    "upgrade-minor-ovn":{
```

Comment:
you created upgrade-minor-ovn jobs for AWS, GCP, and Azure.  If we want all of those, include the platform slug in the name here (`upgrade-minor-ovn-aws`)?  If we are fine with a single platform, drop the jobs for the other platforms?  It should be easy to shift a platform-agnostic job between platforms moving forward.

Reply:
I added all three. thanks for catching that.